implementations/rust/ockam/ockam_app/src/shared_service/tray_menu.rs

```diff
@@ -44,7 +44,7 @@ fn shared_service_submenu(outlet: &OutletStatus) -> SystemTraySubmenu {
         // NOTE: Event handler for dynamic ID is defined in crate::invitations::tray_menu module,
         // and reached via crate::app::tray_menu::fallback_for_id
         submenu = submenu.add_item(CustomMenuItem::new(
-            format!("invitation-create-for-{}", outlet.alias),
+            format!("invitation-create-for-{}", outlet.tcp_addr),
```

Comment:
Wouldn't a TCP address be not necessarily unique? Can I expose the same TCP address and port as two separate relays with different non-overlapping route paths?

Reply:
True... perhaps we should add a new ID ourselves for the desktop app specifically. The problem with the `alias` is that is None by default, so we can't use that either.